SpringProject/src/it/java/com/gradproject2019/conferences/ConferenceControllerIT.java

```diff
@@ -188,23 +193,39 @@ public void shouldReturn200AndEditOnlyNotNullFields() throws URISyntaxException
         });
     }
 
+    private ResponseEntity<ErrorEntity> getSingleConferenceExpectingError(URI uri) {
+        return restTemplate.exchange(uri, GET, null, ErrorEntity.class);
+    }
+
     private ResponseEntity<String> postConference(URI uri, ConferenceRequestDto request) {
         return restTemplate.exchange(uri, POST, new HttpEntity<>(request, passingHeaders), new ParameterizedTypeReference<String>() {
         });
     }
 
+    private ResponseEntity<ErrorEntity> postConferenceExpectingError(URI uri, ConferenceRequestDto request) {
+        return restTemplate.exchange(uri, POST, new HttpEntity<>(request, passingHeaders), ErrorEntity.class);
+    }
+
     private ResponseEntity<String> deleteConference(URI uri) {
         return restTemplate.exchange(uri, DELETE, new HttpEntity<>(passingHeaders), new ParameterizedTypeReference<String>() {
         });
     }
 
+    private ResponseEntity<ErrorEntity> deleteConferenceExpectingError(URI uri) {
+        return restTemplate.exchange(uri, DELETE, new HttpEntity<>(passingHeaders), ErrorEntity.class);
+    }
+
+    private ResponseEntity<ErrorEntity> deleteConferenceExpectingAuthError(URI uri) {
```

Comment:
Small note - this seems very similar to the method above it but for the headers that are either passing or failing - they could be combined into 1 method with an extra param 'headers'? (only if you think it will improve readability in the tests)